inversion_specifications/NAME-RHIME/overview.md

```diff
@@ -0,0 +1,57 @@
+# RHIME inverse model specification
+
+Transport model and inversion setup for the Univeristy of Bristol Regional Hierarchical Inverse Modelling Environment (RHIME). The following applies to all species, unless specified in a species specific file in this folder.
+
+## Transport model
+
+NAME model (version 7.2) developped by the UK Met Office (Muller et al. 2013), a Langrangian particle model that we run backwards in time for each measurement site used in the inversion.
+
+### Scope
+
+- Domain: 10.73°N to 79.06°N and from 97.9°W to 39.38°E.
+- Period: 2018-01-01 to 2023-01-01
+
+### Meteorology
+
+The meteorological fields from the UK Met Office Unified Model are used. The model is run on a grid of horizontal resolution 0.0135° x 0.0135° over the UK (Manning and Redington et al. 2021). Vertically, it uses 57 levels, expanding to 12.2km.
+
+### Model input parameters
+
+The langragian transport model is used with a release of 20000 particles/hour. Particles are released from the measurement altitude ± X m. Particles are tracked for 30 days prior to the measurement, or until they leave the domain.
+
+## Inversion approach
+
+The inversion method is the Hierarchical Bayesian Markov Chain Monte Carlo (HBMCMC) model (as described in Ganesan et al., 2014). The code is in open access on github : https://github.com/openghg/openghg_inversions.
+
+### Model and measurement uncertainties
+
+The measurement are averaged on a four hour basis. We also remove the data for which the PBL height is less than 50m different from the inlet height.
+
+The inversion is done by comparing the observation vector ($Y_{obs}$) to the simulated observations given by:
+
+$$
+Y_{mod} = H_x x + H_{bc}x_{bc} + \epsilon
+$$
+
```

Comment:
@alexdanjou, please can you use Latex math notation in the rest of this section, as I've done here. This should make it easier to follow